monai/data/dataset.py

```diff
@@ -1249,20 +1250,33 @@ class CSVDataset(Dataset):
         transform: transform to apply on the loaded items of a dictionary data.
         kwargs: additional arguments for `pandas.merge()` API to join tables.
 
+    .. deprecated:: 0.8.0
+        ``filename`` is deprecated, use ``src`` instead.
+
     """
 
+    @deprecated_arg(name="filename", new_name="src", since="0.8", msg_suffix="please use `src` instead.")
     def __init__(
         self,
-        filename: Union[str, Sequence[str]],
+        # `src` also can be `pandas.DataFrame` or sequense of `pandas.DataFrame`
```

Comment:
this comment could be appended to line 1262

Reply:
updated in the latest commit.

Thanks.